packages/zwave-js/src/lib/driver/ZWaveOptions.ts

```diff
@@ -59,8 +59,8 @@ export interface ZWaveOptions {
 	 */
 	skipInterview?: boolean;
 
-	/** Allows you to skip user code interview */
-	skipUserCodeInterview?: boolean;
+	/** Allows you to query all user code during the interview of the UserCode CC */
+	queryAllUserCodes?: boolean;
```

Comment:
```suggestion
	/**
	 * Whether all user code should be queried during the interview of the UserCode CC.
	 * Note that enabling this can cause a lot of traffic during the interview.
	 */
	queryAllUserCodes?: boolean; // default: false
```

Reply:
Also, do we still want to merge the interview settings under a common property?